### DISCUSSION TITLE
Microsoft Azure does not pass verification using GitHub

### DISCUSSION THREAD

**Discussion by pavlo-myskov** [community/community#48300] [GitHub Education] [answered]
Hi there,

A few months ago, I received a link to activate my Github Student Developer Pack from my educational institution. I activated it and it works with almost all services except Microsoft Azure.

The main problem is that I'm unable to complete the verification with Microsoft Azure to get student benefits.
When I try to go through the Azure for Students verification by opening this link:
https://signup.azure.com/studentverification?offerType=1
and using GitHub as a verification method I see this:
"
The account you are signed is not yet verified to access offer benefits.
...
We could not confirm your status as a student.
"
I also contacted Microsoft and a Microsoft employee asked me for information to verify my status as a student:
- The **green box** from (https://education.github.com/discount_requests/student_application) confirming my student status as approved.
- The emails section from the same page (What e-mail address do you use for school?), with my verified address visible and selected (marked with the dot on the left).

There is no 'green box' that approves me as a student, and without this, they cannot continue the investigation.
I don't understand why I don't have this confirmation on my page (https://education.github.com/discount_requests/student_application) if I have already activated the Student Pack.

Can you tell me what the issue is?

Also, I opened a ticket to GitHub Support more than a week ago, but no one answered.

**Comment by samus-aran** [comment, marked as answer]
Hi @FlashDrag 👋 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐

We've passed this onto our internal team to investigate but in the meantime we'll respond to your latest Support ticket to help you redeem your Azure benefits. Keep a look out for any updates on your GitHub tickets.

**Reply by pavlo-myskov** [reply]
There was a long investigation process. Thanks to your assistance and confirmation of my verification from GitHub Support, Microsoft finally verified my account

**Reply by VendiConfi** [reply]
please i am having the same issue i have done everything written mails to microsoft even my account github account is linked to microsoft and i am a verfied member of student developers and i still cant acess anything can you please help me too

**Reply by leorohrba** [reply]
me too, the exact same issue

**Reply by Bykathika74** [reply]
I am having the same issue. Idk how to fix this.

**Reply by alidrisy** [reply]
i have the same problem

**Reply by karthikasai1828** [reply]
I have the same issue. I also have verified student developer account but I have issue with Microsoft Azure verifying my same school ID. Can you please anyone help me to fix this issue ASAP.

**Reply by brohithkr** [reply]
I had same issue. It kept saying that my collage domain wasn't registered with them even though my github account is linked and it has been verified as a student.

**Reply by DrMainIT** [reply]
i have the same problem

**Reply by cristiandpto23** [reply]
I have the same issue

**Reply by RahulKumar-007** [reply]
did anyone find any solution

**Reply by Nino678190** [reply]
I have the same issue

**Reply by ghost** [reply]
Same problem

**Reply by IanKidubo** [reply]
i have same issue

**Reply by Bara-BK** [reply]
I have the same issue

**Comment by baietoo** [comment]
Hello, I have a similar situation @samus-aran 
<img width="1432" alt="Screenshot 2023-05-16 at 19 41 58" src="https://github.com/community/community/assets/22468549/8e3bcd2f-63e7-4765-9a72-f8876df54318">
could you please help?

**Reply by VendiConfi** [reply]
hey bro is your problem solved cause ive been stuck exactly here since a week

**Reply by DrMainIT** [reply]
i have the exact same problem did you resolved it ?

**Reply by leenrd** [reply]
is this resolved or what, this sucks, better not give anything than this false promises

**Reply by tharindu-dm** [reply]
having the same issue here

**Comment by saurabhkurve** [comment]
I am facing same issue since a week please let us know what is the reason behind that.
if there is any solution please tell us so that we can continue our azure learning journey.

**Comment by Enniey** [comment]
i am facing the same issue. please help me out

**Comment by Promipi** [comment]
the same

**Comment by siskakumar** [comment]
Same issue

**Comment by satr14washere** [comment]
same issue here

**Comment by jsdhami** [comment]
same issue

**Comment by deviate-dv8** [comment]
same issue

**Comment by afifurrohman-id** [comment]
same issue..

**Comment by kayas881** [comment]
same here

**Comment by lpbedia2792** [comment]
Same issue here
![Screenshot 2024-02-13 091355](https://github.com/community/community/assets/151418997/29aef282-8558-4053-916e-e2a07ef56056)

**Comment by 3IMAD69** [comment]
same here

**Comment by structuraldetailing** [comment]
Same here

**Comment by MissBehavior** [comment]


**Reply by kayas881** [reply]
i contacted them and they sent me a link that worked

**Comment by jaidernunes** [comment]
Same problem here :\

**Comment by satr14washere** [comment]
seems like this is a bug on azure when checking student status

**Comment by Abenu01** [comment]
did anyone solve this?, I am facing same issue 
![image](https://github.com/user-attachments/assets/a42383a3-e322-44df-afba-f78e12e5f5f2)

**Comment by abdul-irfan-k** [comment]
i am also facing same issue

**Comment by Saharsh1223** [comment]
same issue

**Reply by itsmehaelas** [reply]
same issue

**Comment by Saharsh1223** [comment]
i've contacted microsoft support and i'll let y'all know if mine got fixed or not.

**Reply by Saharsh1223** [reply]
> i've contacted microsoft support and i'll let y'all know if mine got fixed or not.

they've managed to resolve the issue a week before i'm messaging this, i think it's the only way to resolve this issue. they gave me a 25 digit verification code, allowing me to get my azure students subscription

**Comment by itsmehaelas** [comment]
thanks you so much

**Comment by itsmehaelas** [comment]
how did you get the 25 digit code?

**Reply by Saharsh1223** [reply]
i contacted microsoft support
they then gave me a 25 digit one time use verification code to pass azure student verification

**Reply by Hiroohamada** [reply]
Can you provide the link to microsoft support

**Reply by ghost** [reply]
> i contacted microsoft support they then gave me a 25 digit one time use verification code to pass azure student verification

can you send the link to microsoft support please

**Comment by itsmehaelas** [comment]
can you provide me the steps?

**Reply by Saharsh1223** [reply]
there's no steps involved. the only step is contact microsoft support and proceed from there.

**Comment by itsmehaelas** [comment]
I already contact microsoft and they dont provide me a 25 digit code they
only provide steps on how to login and still keeps me getting the same
error message

**Comment by LamonLind** [comment]
Same problem
![Uploading Screenshot_20240929_155625.jpg…]()

**Comment by aitnacer-nabil** [comment]
same problem

**Comment by vijaykr338** [comment]
same issue

**Comment by ghost** [comment]
same issue

**Comment by saurabhkurve** [comment]
Contact to Microsoft support they will help you.

**Reply by ghost** [reply]
Can you give me the link of the support please ? Did they solve the problem for you ?

**Comment by saurabhkurve** [comment]
Yes, they resolved the issue by providing me with a 25-digit verification
code. When attempting to create an account with Azure, there is an option
available for customer support.

**Reply by saurabhkurve** [reply]
yes they did. Just try create account with azure you will get the link there.

**Reply by Aditya8782** [reply]
Did they ask you for voter id and all other stuff?? What other things did they ask apart from student id  and linkedin profile??

**Comment by TenZ07** [comment]
Facing the same issue.

**Comment by satr14washere** [comment]
i tried to create a support ticket but the azure website just suggest me answers that doesnt work

**Reply by Aditya8782** [reply]
You have to create the ticket , and then support will contact you through mails and i guess from there on this conversation will proceed. Doing the same thing myself. Lets see how much time it takes.

**Comment by LamonLind** [comment]
They ask only some screenshots and the email that you have signed up with
GitHub education.
